### ISSUE TITLE
[Bug] Cannot find module 'vite-plugin-ssr/plugin' or its corresponding type declarations.ts(2307)

### ISSUE THREAD

**Issue by harrytran998** [brillout/vite-plugin-ssr#133]
When I import this plugin in `vite.config.ts` ==> Display error like title.
![image](https://user-images.githubusercontent.com/24296018/129464804-dbdcfa6e-04fd-4bc2-9ec4-7dce3234f929.png)

But it's worked! Just feel annoying when I open this file and see that error 😁. Hope you can fix tha

**Comment by wobsoriano** [comment]
Just add `index.d.ts` at the root folder with `declare module 'vite-plugin-ssr/plugin'` for now hehe

**Comment by harrytran998** [comment]
Thank you <3. I think your team can make this better with an index.d.ts in this package

```
declare module 'vite-plugin-ssr/plugin' {
  import { Plugin } from 'vite'
  function vitePluginSsr(options?: SsrOtpions): Plugin
  export default vitePluginSsr
}
```

And add this in `package.json` = `"types": "./index.d.ts"`.

When do this + a note in readme add `vite-plugin-ssr/plugin` on `types` in `tsconfig.json` --> More happier for everyone 🤣

**Comment by brillout** [comment]
This is actually already fixed on master. Going to release a new minor soon.

Thanks for the opening this ticket and let me know if there is anything else.

**Comment by brillout** [comment]
Fix released in `0.2.9`.
